frontend/database/modules/datasets.js

```diff
@@ -606,18 +616,31 @@ const actions = {
             labels,
           });
 
-          newLabels.forEach(
-            (newLabel) =>
-              isLabelTextExistInGlobalLabel(datasetId, newLabel) ||
-              insertNewGlobalLabel({ datasetId, newLabel })
-          );
+          // NOTE - delete then init globalLabels to update shortcuts by labels
+          deleteAllGlobalLabelModel();
+          labels.forEach((newLabel) => {
+            upsertNewGlobalLabel({
+              datasetId,
+              datasetName,
+              newLabel,
+              isActivate: false,
+              isSavedInBack: true,
+            });
+          });
         } catch (err) {
-          throw new Error("Error on adding new labels");
+          const { response } = err;
+          if (response === TYPE_OF_FEEDBACK.NOT_ALLOWED_TO_UPDATE_LABELS) {
+            console.log(
+              "The user is not allowed to update dataset settings labels"
+            );
```

Comment:
Do we need this `console.log` for any reason?

Reply:
This is only for information for testing. We can remove later